apps/vs-code-designer/src/package.json

```diff
@@ -67,6 +67,15 @@
           "dark": "assets/dark/CreateNewProject.svg"
         }
       },
+      {
+        "command": "azureLogicAppsStandard.createNewCodeProject",
+        "title": "Create New Code Project...",
```

Comment:
"Create new logic app workspace"

Reply:
@ecfan thoughts?

Reply:
```suggestion
        "title": "Create new workspace for code projects",
```

Reply:
> @samikay101 & @rohithah: Same question for the same string in `apps/vs-code-designer/src/app/commands/createNewCodeProject/createNewCodeProject.ts`:
> 
> How does deployment work? Do you deploy specific parts of the workspace, such as just the projects, or can deployment include other artifacts in the workspace?

we only deploy the Logic App folder, other parts will get pushed into source repo and used to build the final package as well as running locally (e.g., debugging)